packages/core/admin/admin/src/translations/en.json

```diff
@@ -481,26 +481,26 @@
   "components.FilterOptions.FILTER_TYPES.$notNull": "is not null",
   "components.FilterOptions.FILTER_TYPES.$null": "is null",
   "components.FilterOptions.FILTER_TYPES.$startsWith": "starts with",
-  "components.Input.error.attribute.key.taken": "This value already exists",
+  "components.Input.error.attribute.key.taken": "The value already exists",
   "components.Input.error.attribute.sameKeyAndName": "Can't be equal",
   "components.Input.error.attribute.taken": "This field name already exists",
   "components.Input.error.contain.lowercase": "Password must contain at least one lowercase character",
   "components.Input.error.contain.number": "Password must contain at least one number",
   "components.Input.error.contain.uppercase": "Password must contain at least one uppercase character",
-  "components.Input.error.contentTypeName.taken": "This name already exists",
+  "components.Input.error.contentTypeName.taken": "The name already exists",
   "components.Input.error.custom-error": "{errorMessage} ",
-  "components.Input.error.password.noMatch": "Passwords do not match",
-  "components.Input.error.validation.email": "This is an invalid email",
-  "components.Input.error.validation.json": "This doesn't match the JSON format",
+  "components.Input.error.password.noMatch": "The passwords do not match",
+  "components.Input.error.validation.email": "The value is an invalid email",
+  "components.Input.error.validation.json": "The value is invalid JSON",
   "components.Input.error.validation.lowercase": "The value must be a lowercase string",
-  "components.Input.error.validation.max": "The value is too high.",
-  "components.Input.error.validation.maxLength": "The value is too long.",
-  "components.Input.error.validation.min": "The value is too low.",
-  "components.Input.error.validation.minLength": "The value is too short.",
-  "components.Input.error.validation.minSupMax": "Can't be superior",
-  "components.Input.error.validation.regex": "The value does not match the regex.",
-  "components.Input.error.validation.required": "This value is required.",
-  "components.Input.error.validation.unique": "This value is already used.",
+  "components.Input.error.validation.max": "The value is larger than the maxmimum",
+  "components.Input.error.validation.maxLength": "The value is longer than the maximum",
+  "components.Input.error.validation.min": "The value is smaller than the minimum",
+  "components.Input.error.validation.minLength": "The value is shorter than the minimum",
+  "components.Input.error.validation.minSupMax": "The value can not be superior",
```

Comment:
I think it should be "cannot" :)

Reply:
Updated in 304901ed7ef7d061fe7860ef36db60aa78ac66b2